src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/TableMathfuncs.txt

```diff
@@ -34,4 +34,49 @@ Blank()
 2
 
 >> Average([1, 2, 1/0, 4, 5], Value)
-#Error
\ No newline at end of file
+#Error
+
+>> VarP(1, 3, 5)
```

Comment:
We're moving towards putting function-specific tests in their own .txt file. Makes searching easier, and it becomes kind of a spec for the function.

Reply:
Moved the VarP and StdevP tests into their own files, but left sum and average alone as I noticed they have unused tests in the NotYetReady test-case folder.

Reply:
Thanks. The existing ones will get cleaned up later, no need to worry about that for now. Just trying to make sure new ones are sensible.